packages/superset-ui-chart/src/components/SuperChart.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react';
+import { createSelector } from 'reselect';
+import getChartComponentRegistry from '../registries/ChartComponentRegistrySingleton';
+import getChartTransformPropsRegistry from '../registries/ChartTransformPropsRegistrySingleton';
+import ChartProps from '../models/ChartProps';
+import createLoadableRenderer, { LoadableRenderer } from './createLoadableRenderer';
+
+const IDENTITY = (x: any) => x;
+
+const EMPTY = () => null;
+
+/* eslint-disable sort-keys */
```

Comment:
Might be worth-while disabling `sort-keys` in our `build-config`.

Reply:
yeah I have mixed feelings on that rule, it's a pain sometimes esp when you want to logically group things like this. I'll make the change in build-config.

Reply:
👍